src/components/ResumePage/element/ResumePreview.js

```diff
@@ -67,6 +69,12 @@ const ResumePreview = () => {
                 {about.email ? about.email : "HarryPotter@hogwarts.edu"}
               </Text>
             </HStack>
+            <HStack spacing={1}>
+              <BsGlobe />{" "}
+              <Text as="a" href={about.website}>
+                Website
```

Comment:
can you refrain from applying the hover color change effect on both the Website and the Linkedin Text. Furthermore, if possible, could you kindly use a different icon to replace the globe icon?